shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -459,12 +461,29 @@ @implementation FlutterEngine {
 
   // Proxy to allow plugins, channels to hold a weak reference to the binary messenger (self).
   FlutterBinaryMessengerRelay* _binaryMessenger;
+
+  // Map from ViewId to vsync waiter. Note that this is modified on main thread
+  // but accessed on UI thread, so access must be @synchronized.
+  NSMapTable<NSNumber*, FlutterVSyncWaiter*>* _vsyncWaiters;
 }
 
 - (instancetype)initWithName:(NSString*)labelPrefix project:(FlutterDartProject*)project {
   return [self initWithName:labelPrefix project:project allowHeadlessExecution:YES];
 }
 
+static void SetThreadPriority(FlutterThreadPriority priority) {
+  if (priority == kDisplay || priority == kRaster) {
+    pthread_t thread = pthread_self();
+    sched_param param;
+    int policy;
+    if (!pthread_getschedparam(thread, &policy, &param)) {
+      param.sched_priority = 47;  // Same priority as main thread.
```

Comment:
Consider extracting this out as a constant to make it a little more self-documenting.